starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  /* border: 1px solid red; */
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif
+}
+
+.wrapper {
+  width: 900px;
+  margin: 0 auto;
+}
+
+.header {
+  height: 60px;
+  line-height: 40px;
+}
+
+.header .logo {
+  height: 40px;
+  float: left;
+  padding: 10px 15px;
+}
+
+.logo-jump:hover {
+  animation: bounce 1.2s ease-out;
+}
+
+.header .nav {
+  float: right;
+  padding: 10px 15px;
+}
+
+.header .nav ul {
+  list-style-type: none;
+}
+
+.header .nav li {
+  display: inline;
+  padding: 0 5px;
+  color: grey;
+  font-size: .9em;
+  font-weight: lighter;
+}
+
+.banner {
+  width: 900px;
```

Comment:
if you use vh & vw, you can change the background size with cover
```css
.banner {
	width: 100vw;
	background: url("../images/landing.jpg") no-repeat center center;
	background-size: cover;
	height: 80vh;
	text-align: center;
}
```
result:
![image](https://user-images.githubusercontent.com/39528473/40471396-dd90cbe6-5f36-11e8-8262-148876371f03.png)